_config.yml

```diff
@@ -10,7 +10,7 @@ url: 'https://wott.io'
 
 # Website info
 title: Web of Trusted Things
-description: WoTT is an automated, open-source public key infrastructure for connected hardware
+description: Seamless security audit of linux systems for devops teams.
```

Comment:
'devops' -> DevOps

Reply:
Fixed in https://github.com/WoTTsecurity/wott-io/pull/171/commits/8da1167f790c9a5ea2cb01dcf7ddbb04aa453db7